packages/components/src/components/button/_button.scss

```diff
@@ -121,19 +122,31 @@
       background: transparent;
       border-color: transparent;
 
-      .#{$prefix}--btn__icon {
+      .#{$prefix}--btn__icon path {
         fill: $disabled;
       }
     }
 
     &.#{$prefix}--btn--sm {
-      padding: 0.375rem 1rem;
+      padding: $button-padding-ghost-sm;
     }
   }
 
+  .#{$prefix}--btn--icon-only {
+    @include tooltip--icon;
+  }
+
+  .#{$prefix}--btn--icon-only--top {
+    @include tooltip--icon--top;
+  }
+
+  .#{$prefix}--btn--icon-only--bottom {
+    @include tooltip--icon--bottom;
+  }
+
   .#{$prefix}--btn--icon-only,
   .#{$prefix}--btn--sm.#{$prefix}--btn--icon-only {
-    padding-right: rem(15px);
+    padding-right: rem(13px);
```

Comment:
Just wanted to make sure the change here is still intentional.

Reply:
yes this change is intentional!